vspace/src/main/java/edu/asu/diging/vspace/core/references/DefaultMetadata.java

```diff
@@ -0,0 +1,213 @@
+package edu.asu.diging.vspace.core.references;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.exception.ReferenceMetadataEncodingException;
+import edu.asu.diging.vspace.core.model.impl.Reference;
+
+@Service
+@PropertySource({"classpath:config_reference.properties"})
+public class DefaultMetadata implements ReferenceMetadataProvider {
```

Comment:
Also, this is the COinS metadata provider, isn't it? There isn't really a default metadata embedded (the default is for how the citations are displayed on the website, not how it can be harvested by tools like Zotero).